src/components/text/text.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import './text.css';
+
+import data from '../../data';
+
+const Text = ({ pageName, controlHeader }) => {
+  const pageData = data[pageName];
+  let dataKey = '';
+
+  switch (controlHeader) {
+    case 'what':
+      dataKey = 'What?';
+      break;
+    case 'how':
+      dataKey = 'How?';
+      break;
+    case 'more-information':
+      dataKey = 'More Information';
+      break;
+    default:
+      break;
+  }
+
+  const textData = pageData[dataKey];
+  let text;
+  
+  if (Array.isArray(textData)) {
```

Comment:
it's more clear if you move the logic to a function
`const text = getText(textData)`

Reply:
I see.

Reply:
Done